n2y/errors.py

```diff
@@ -26,6 +26,14 @@ class UseNextClass(Exception):
     pass
 
 
+class UseNextType(Exception):
```

Comment:
I was thinking about this; what's the benefit of creating a new exception class here, instead of using the existing `UseNextClass` exception?

Reply:
It just created it for specificity, you'd know there was an error in instantiating a class from a pandoc type if this error was thrown and an error in instantiating a class from notion if the other was. It's deprecated now, as I access the classes directly.